src/mqt/predictor/rl/Predictor.py

```diff
@@ -61,13 +62,13 @@ def compile_as_predicted(
             action_item = self.env.action_set[action]
             used_compilation_passes.append(action_item["name"])
             obs, reward_val, terminated, truncated, info = self.env.step(action)
-            self.env.state._layout = self.env.layout
+            self.env.state.layout = self.env.layout
```

Comment:
Yes, because we rely in `MQT Bench v1.0.0` which relies on `pytket-qiskit v0.40.0` which relies on `qiskit v0.43.0` where the public attribute was introduced.